src/commands/test/test_environment_exceptions.py

```diff
@@ -10,6 +10,11 @@ class MissingExceptReportedException(ReportedException):
     pass
 
 
+class TestNotFailedReportedException(ReportedException):
```

Comment:
This naming is not really intuitive, can we skip `ReportedException` in the names?

Reply:
Removed `Reported` part, but left `Exception` part.